charts/azuremonitor-containers/values.yaml

```diff
@@ -33,6 +33,10 @@ omsagent:
   # chance to build pod for the node and give it to the scheduler)
   # Should be some number greater than default (0)
   priority: 10
+  
+  # This used for running agent pods in test mode and setting true makes the agent pods running in test mode 
```

Comment:
[pedantic] This is really wordy, consider:
# This used for running agent pods in test mode. 
# if set to true additional agent workflow logs will be emitted which are used for e2e and conformance testing

Reply:
thanks for the suggestion. fixed :)